client/src/client/user_interface/__init__.py

```diff
@@ -113,45 +110,49 @@ def __init__(
             ),
             StartupState.PROMPT_COLOR: InputState(
                 callback_name="set_identity_color",
-                title=" 2 - Set color",
+                title=f"Startup {StartupState.PROMPT_COLOR.value} - Set color",
                 prompt="Please input the color you want to "
                 "use for your username, "
                 "it can be changed later.",
                 validation=self.validate_color,
             ),
             StartupState.PROMPT_PASSWORD: InputState(
                 callback_name="set_password",
-                title=" [3] - [Optional] Set password",
+                title=f"Startup [{StartupState.PROMPT_PASSWORD.value}]"
+                " - [Optional] Set password",
                 prompt="If you want to have additional protection, input a "
                 "password for your account. Password cannot contain any "
                 "whitespace characters and if provided it should be at  "
                 "least 6 characters long.",
                 validation=self.validate_password,
             ),
+            "password": InputState(
+                callback_name="set_password",
+                title="Input password",
+                prompt="Please input your password."
+                "Input '/' to enter password recovery mode.",
+                validation=self.validate_password,
+            ),
         }
-        """Startup states in from (callback_name, tile, prompt, validation)"""
+        """Prompt states in from (callback_name, tile, prompt, validation)"""
 
         self.last_closed_tile: List[Tile] = []
 
-        # launch StartupTile
-        self.first_startup = first_startup
+        self.prompt_tile: Optional[PromptTile] = None
 
-        self.startup_tile: Optional[PromptTile] = None
-        if not self.first_startup:
-            self.startup_tile = self.view.add_startup_tile(
-                prompt_text="Please enter password. Input '/'"
-                " to enter password recovery mode.",
-                title="Startup",
-                validation_function=self.validate_password,
-            )
-        else:
-            self.startup_tile = None
-            self.first_startup_prompt(
-                state=StateMachine(StartupState, state=0).state
-            )
+        self.loop.run_until_complete(self.create_queue())
```

Comment:
Can't this be called directly? Queue creation is itself synchronous

Reply:
It can, but this is more unambgous. asyncio.Queue binds itself to the main loop in thread, unless you invoke it from within the loop. I just don't wanna debug for 2 hours just because queue has problems

Reply:
Ok, I was not aware of this binding mechanism